### DISCUSSION TITLE
How do you guys manage `border-radius`, spacing etc?

### DISCUSSION THREAD

**Discussion by CornerSyrup** [material-components/material-web#5245] [Polls]
If I recall correctly, MD2 provide round corner tokens and spacing tokens (in fact, I didn't use MD2 web before, just Vuetify). Now, Material Theme Builder don't generate those spacing token anymore, which means I need to figure out with me brilliant mind. So, here's my question,

**Comment by datvm** [comment]
I am still using Bootstrap CSS because of this. If MD3 provides one I am happy to switch.
